test/images/agnhost/webhook/pods.go

```diff
@@ -77,6 +80,25 @@ func admitPods(ar v1.AdmissionReview) *v1.AdmissionResponse {
 }
 
 func mutatePods(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	selector := func(pod *corev1.Pod) bool {
+		return pod.Name == "webhook-to-be-mutated"
+	}
+	return applyPodPatch(ar, selector, podsInitContainerPatch)
+}
+
+func mutatePodsSidecar(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	selector := func(pod *corev1.Pod) bool {
+		for _, container := range pod.Spec.Containers {
+			if container.Name == "webhook-added-sidecar" {
+				return false
+			}
+		}
+		return true
+	}
+	return applyPodPatch(ar, selector, podsSidecarPatch)
+}
+
+func applyPodPatch(ar v1.AdmissionReview, podSelector func(*corev1.Pod) bool, patch string) *v1.AdmissionResponse {
```

Comment:
`podSelector` is a confusing name for what this does... `shouldPatchPod` or something would be clearer

Reply:
Done.